tests/spread/cross-compile/environment/snaps/build-for-multi-arch/snap/snapcraft.yaml

```diff
@@ -12,12 +12,15 @@ architectures:
 
 environment:
   SNAPCRAFT_ARCH_TRIPLET: ${SNAPCRAFT_ARCH_TRIPLET}
-  SNAPCRAFT_ARCH_BUILD_FOR: ${SNAPCRAFT_ARCH_BUILD_FOR}
-  SNAPCRAFT_ARCH_TRIPLET_BUILD_FOR: ${SNAPCRAFT_ARCH_TRIPLET_BUILD_FOR}
   SNAPCRAFT_ARCH_BUILD_ON: ${SNAPCRAFT_ARCH_BUILD_ON}
   SNAPCRAFT_ARCH_TRIPLET_BUILD_ON: ${SNAPCRAFT_ARCH_TRIPLET_BUILD_ON}
   SNAPCRAFT_TARGET_ARCH: ${SNAPCRAFT_TARGET_ARCH}
 
 parts:
   my-part:
     plugin: nil
+    override-pull:
+      echo ${SNAPCRAFT_ARCH_TRIPLET}
```

Comment:
Maybe put a test command here to ensure it's empty as we expect?

Reply:
`SNAPCRAFT_ARCH_TRIPLET` is a confusing variable that will always exist and not be empty.  Sometimes it's `build-for` and sometimes `build-on`, determined by convoluted business logic.

I didn't change that variable but I did add it to these tests to help catch regressions in the future.